### DISCUSSION TITLE
Units for gas values in UserHorizontalTransmission

### DISCUSSION THREAD

**Discussion by photon-rain** [space-physics/lowtran#24] [Q&A]
In the user defined files the value given for the water content of the atmosphere is 93.96. I have been unable to determine what unit this is given in. Line 1145 of ``lowtran7.f`` indicates that this is in mb, however this value is too large to be mb or any of the other units defined for use in the Lowtran manual. Could you please clarify what the unit for this quantity is?

**Comment by georgePadolsey** [comment]
The unit for this quantity is given in [relative humidity](https://en.wikipedia.org/wiki/Humidity#Relative_humidity) (RH), which is %. This is currently hard-set in the Fortran code. See Line 1145-1155.

<details><summary>Line 1145-1155</summary><p>

```fortran

junit(1)=10  !10: units of total pressure: millibar
junit(2)=10  !10: units of temperature: Kelvin
junit(3)=17  !17: H20 as relative humidity (%)

! now set WMOL units
do ipy2 = 4,15
   junit(ipy2) = 14   !FIXME hard set to PARTIAL PRESSURE for Local Meterological experiments. Can be upgraded.
enddo

do ipy2 = 1,12
   WMOL(ipy2) = WMOLpy(ipy2)
enddo
 ```
</p>
</details>

In particular: `junit(3) = 17`: the unit for H20 (WMOL[3]) is set to H [^1]. See table 11A on page 32 of the Lowtran user manual to check this.

<details><summary>Screenshot of Lowtran User Manual Table 11A</summary>
<img width="754" alt="Screenshot of Lowtran User Manual Table 11A" src="https://user-images.githubusercontent.com/2631084/141177704-1e6be912-a94f-4b37-8979-a89f4a3592e0.png">
</details>

Hope this helps :) 

[^1]: The way the numbering is done is 1-9, A, B, ... , G, H,... hence 17 = H
